src/dynamic_boundary_conditions/hirds_rainfall_data_from_db.py

```diff
@@ -78,10 +78,16 @@ def get_each_site_rainfall_data(
         query = f"""SELECT * FROM {rain_table_name}
         WHERE site_id='{site_id}' AND rcp='{rcp}' AND time_period='{time_period}' AND ari={ari};"""
         rain_data = pd.read_sql_query(query, engine)
+        # drop category column
+        rain_data.drop(["category"], axis=1, inplace=True)
     else:
         query = f"""SELECT * FROM {rain_table_name}
         WHERE site_id='{site_id}' AND rcp IS NULL AND time_period IS NULL AND ari={ari};"""
-        rain_data = pd.read_sql_query(query, engine).head(1)
+        rain_data = pd.read_sql_query(query, engine)
+        # filter for historical data
+        rain_data.query("category == 'hist'", inplace=True)
+        # drop category column
+        rain_data.drop(["category"], axis=1, inplace=True)
```

Comment:
You could just drop the category column once at the end instead of twice.
Or maybe we don't need to drop the column at all?

Reply:
avoided dropping the column a all. it can always be filtered out later if needed.